docs/tutorial/snapcraft-guide.md

```diff
@@ -0,0 +1,181 @@
+2) Using an already created `.deb` package
+3) Using [`electron-forge`](electron-forge) or
+   [`electron-builder`](electron-builder), both tools that come with `snap`
+   support out of the box (not further documented here, please see those
+   frameworks for further guidance)
+
+In both cases, you will need to have the `snapcraft` tool installed. We
+recommend building on Ubuntu 16.04 (or the current LTS).
+
+```sh
+snap install snapcraft --classic
+```
+
+While it _is possible_ to install `snapcraft` on macOS using Homebrew, you are
+less likely to encounter issues when running `snapcraft` on an actual Linux
+distribution.
+
+# Using `electron-installer-snap`
+
+The module works like `electron-winstaller` and similar modules in that its
+scope is limited to building snap packages. You can install it with:
+
+```sh
+npm install --save-dev electron-installer-snap
+```
+
+## Step 1: Package Your Electron Application
+
+Package the application using [electron-packager][electron-packager] (or a
+similar tool). Make sure to remove `node_modules` that you don't need in your
+final application, since any module you don't actually need will just increase
+your application's size.
+
+The output should look roughly like this:
+
+```text
+├── Ghost.exe
+├── LICENSE
+├── content_resources_200_percent.pak
+├── content_shell.pak
+├── d3dcompiler_47.dll
+├── ffmpeg.dll
+├── icudtl.dat
+├── libEGL.dll
+├── libGLESv2.dll
+├── locales
+│   ├── am.pak
+│   ├── ar.pak
+│   ├── [...]
+├── natives_blob.bin
+├── node.dll
+├── resources
+│   ├── app
+│   └── atom.asar
+├── snapshot_blob.bin
+├── squirrel.exe
+└── ui_resources_200_percent.pak
+```
+
+## Step 2: Running electron-installer-snap
+
+From a terminal that has `snapcraft` in its `PATH`, run `electron-installer-snap`
+with the only required parameter `--out`, which is the location of your packaged
+Electron application created in the first step.
+
+```sh
+npx electron-installer-snap --src=out/myappname-linux-x64
+```
+
+If you have an existing build pipeline, you can use `electron-installer-snap`
+programmatically. For more information, see the API docs.
+
+```js
+const snap = require('electron-installer-snap')
+
+snap(options)
+  .then(snapPath => console.log(`Created snap at ${snapPath}!`))
+```
+
+# Using an Existing Debian Package
+
+Snapcraft is capable of taking an existing `.deb` file and turning it into
+a `.snap` file. The creation of a snap is configured using a `snapcraft.yaml`
+file that describes the sources, dependencies, description, and other core
+building blocks.
+
+## Step 1: Create a Debian Package
+
+If you do not already have a `.deb` package, using `electron-installer-snap`
+might be an easier path to create snap packages. However, multiple solutions
+for creating Debian packages exist, including [`electron-forge`](electron-forge),
+[`electron-builder`]() or [`electron-installer-debian`](electron-installer-debian).
```

Comment:
did you forget the link for `electron-builder`, or is this clever markdown magic?